lib/httparty/request/body.rb

```diff
@@ -46,26 +46,16 @@ def generate_multipart
         multipart += "--#{boundary}--\r\n"
       end
 
-      def has_file?(hash)
-        hash.detect do |key, value|
-          if value.respond_to?(:to_hash) || includes_hash?(value)
-            has_file?(value)
-          elsif value.respond_to?(:to_ary)
-            value.any? { |e| file?(e) }
-          else
-            file?(value)
-          end
+      def file?(value)
```

Comment:
I feel like `#has_file?` is better a naming for this method, anyway.
You still ask whether the `params` have a file or not. You're right that it can be `file`, but it can also be hash, array and `#has_file?` is a more inclusive naming

Reply:
yeah, you are right. I changed here 5a778ed2eac655ecebc0e2b134e9d53a29026aa9